releasenotes/notes/auto-adding-baggage-to-span-tags-0ec127de76ca97bf.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    baggage: Adds support for automatically adding baggage key-value pairs to span tags. Baggage items from incoming HTTP headers are attached to spans as tags with a `baggage.` prefix
+    The `DD_TRACE_BAGGAGE_TAG_KEYS` configuration allows users to specify a comma-separated list of baggage keys for span tagging, by default the value is set to `usr.id,account.id,session.id`.
+    Setting it to an empty value disables baggage tagging."
```

Comment:
We should probably show what an empty value looks like somewhere, and should add docs in configuration.rst